Manhattan-Pizza-and-Pub.md

```diff
@@ -0,0 +1,19 @@
+# Manhattan Pizza & Pub
+
+##Address:## 167 Main St, Burlington, VT 05401
+
+*Hours:* 
+_Friday  	11:30AM–2AM
+Saturday	11:30AM–2AM
+Sunday  	12:30PM–12AM
+Monday  	8PM–1AM
+Tuesday 	4:30PM–2AM
+Wednesday	4:30PM–2AM
+Thursday	4:30PM–2AM_
+
+
+Menu: <manhattanpizzaandpubvt.com>
+
+
+
+Dylan Morosini
```

Comment:
please [remove your name](https://martinfowler.com/bliki/CodeOwnership.html) and make a new commit :-)